Views/Items/FormOutputTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@using Etch.OrchardCore.Workflows.FormOutput.Workflows.ViewModels;
+@model FormOutputTaskViewModel
+
+<fieldset class="form-group" asp-validation-class-for="Prefix">
+    <label asp-for="Prefix">@T["Prefix"]</label>
+    <input asp-for="Prefix" class="form-control" autofocus="autofocus" />
```

Comment:
Sure you want this to have autofocus?

Reply:
Ah yeah both of these are because I'm rushing a bit, copying from other places. Will remove them.